massdriver.yaml

```diff
@@ -153,12 +155,60 @@ params:
         replicas_per_primary:
           type: integer
           title: Replicas
-          description: Number of read replicas per primary node. When the primary VM becomes unavailable, the replica detects that and takes over as the new primary automatically. This setting cannot be changed.
+          description: "Number of read replicas per primary node. When the primary VM becomes unavailable, the replica detects that and takes over as the new primary automatically. **This cannot be changed after deployment**."
           $md.immutable: true
           enum:
             - 1
             - 2
             - 3
+        persistence:
```

Comment:
I was able to solve this ridiculously time consuming problem by merging (near) all possibilities within a single param. This means that conditional Terraform is not dependent on schema dependencies, which is where I was stuck on for some of yesterday and all of today.